mage/releases/publish.go

```diff
@@ -217,3 +231,35 @@ func listFiles(dir string) []string {
 
 	return names
 }
+
+func generateChecksumFile(path string) (string, error) {
+	if filepath.Ext(path) == "sha256sum" {
```

Comment:
Wouldn't the extension be ".sha256sum"? 🤔

Reply:
You are absolutely right!! Thanks for catching it